kubernetes/samples/scripts/create-fmw-infrastructure-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -142,6 +142,11 @@ domainHomeImageBase: container-registry.oracle.com/middleware/fmw-infrastructure
 # which uses WDT, instead of WLST, to generate the domain configuration.
 domainHomeImageBuildPath: ./docker-images/OracleFMWInfrastructure/samples/12213-domain-home-in-image
 
+# Uncomment to customize the WebLogic Deploy Tool (WDT) version used by the WDT
+# "domain home in image" Docker image sample in the Docker images project. See
+# "domainHomeImageBuildPath". Set to "LATEST" to use the latest version.
+# wdtVersion: LATEST
```

Comment:
Why would we not want LATEST to be the default?

Reply:
This change is already merged.

This change implicitly defaults to 1.9.1 since that's what we used in all of the other samples already - presumably because this helps make sure the samples are stable/reproducable.  (I don't document the default here because I assume that'll change over time.)